src/data/events.js

```diff
@@ -2,7 +2,7 @@ import { writable } from 'svelte/store'
 import { GET, POST, PUT } from './api'
 import { onClear } from './storage'
 
-export const events = writable([])
+export const events = writable(null)
```

Comment:
This is a creative solution. If I understand correctly, `null` means it hasn't been initialized yet, so the consuming code can tell the difference between uninitialized (`null`) and actually empty (`[]`). An observation: later calls to `loadEvents` would need to set `events` back to `null` to make this fully correct. That's pretty much the purpose of `loading` so I don't think this is necessary.

Reply:
I had to remove this because some other parts of the site weren't working because of this change. Instead, I made the `loading` variable be initialized to `true` so that it doesn't go from `false` to `true` and then back to `false`.